Firebase/InstanceID/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 2020-08 -- 4.5.2
```

Comment:
Should be 4.6.0 with the new functionality

Reply:
There's no public API change, I guess this is for better version managing?

Reply:
Minor version updates aren't just for public API changes - they should be used for any changes that go beyond bug fixes.  See https://semver.org/